internal/impl/kafka/output_kafka_franz.go

```diff
@@ -40,6 +41,7 @@ This output is new and experimental, and the existing ` + "`kafka`" + ` input is
 		Field(service.NewInterpolatedStringField("key").
 			Description("An optional key to populate for each message.").Optional()).
 		Field(service.NewStringAnnotatedEnumField("partitioner", map[string]string{
+			"murmur2_hash": "Kafka's default hash algorithm that uses a 32-bit murmur2 hash of the key to compute which partition the record will be on.",
```

Comment:
Might want to add `uniform_bytes` for the UniformBytesPartitioner, I recommend this one over all the rest now